src/components/title/Title.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Image from "next/image";
+import banner from "@/public/home/banner.svg";
+
+interface TitleNameProps {
+  text: string;
+}
+
+const Title: React.FC<TitleNameProps> = ({ text }) => {
+  return (
+    <div className="relative mt-1 flex items-center justify-center">
+      <div className="mr-4 mt-4 h-[.7vh] w-[20vw] bg-[#6F7765]" />
```

Comment:
```suggestion
      <div className="mr-4 mt-4 h-[.8vh] w-[20vw] bg-csa-green-100" />
```
use tailwind config colors ex: bg-csa-green-100